test/lib/validation/schema/specVersion/kind/project/application.js

```diff
@@ -291,223 +291,6 @@ SpecificationVersion.getVersionsForRange(">=2.0").forEach(function(specVersion)
 			}
 		]);
 	});
-
-	test(`Invalid builder configuration (specVersion ${specVersion})`, async (t) => {
```

Comment:
From the diff I can't tell why this test is not valid in specVersion 3.2 anymore. Could you elaborate?

Reply:
Yeah, sure!

The test would fail for specVersion 3.2 becuase of the new property `depCache` that needs to be in the list bellow:
```js
{
				dataPath: "/builder/bundles/1/bundleDefinition/sections/1/mode",
				keyword: "enum",
				message: "should be equal to one of the allowed values",
				params: {
					allowedValues: ["3.1", "3.0", "2.6", "2.5", "2.4"].includes(specVersion) ? [
						"raw",
						"preload",
						"require",
						"provided",
						"bundleInfo"
					] : [
						"raw",
						"preload",
						"require",
						"provided"
					]
				}
			},
```
So, I moved this test to specVersions 2.0 - 3.1 (hoping for a better diff here 😅 ) and copied it with the correct values for 3.2.
Otherwise, this ternary operator would have bocome more complex.